packages/polyfill/source/NamedNodeMap.ts

```diff
@@ -60,7 +67,11 @@ export class NamedNodeMap {
         if (prev) prev[NEXT] = attr[NEXT];
         if (this[CHILD] === attr) this[CHILD] = attr[NEXT];
         updateElementAttribute(ownerElement, attr.name, attr.value, null);
-        hooks.removeAttribute?.(ownerElement as any, name, namespaceURI);
+        ownerElement[OWNER_DOCUMENT].defaultView[HOOKS].removeAttribute?.(
```

Comment:
What do you think about adding a `HOOKS` getter to `Node` as well, that internally does the `[OWNER_DOCUMENT].defaultView[HOOKS]` accessing?

Reply:
That's a good call. Should clean things up.